### PR TITLE
Unsupported file format

### PR DESCRIPTION
**Pull request by lucierabahi**
- I created a stateless `UnsupportedFileFormat.component `
- Added a background image which wasn't mentioned on the Trello card but we talk about it with Neha  and Arnas. I picked up this image but it can be changed if you guys don't like it.
- Created a `mixins.css` file in the `globalScss` folder with mixins for the overlay and for flexbox settings, as I think both will often be used
- Button isn't SVG. I wrote this criteria but I'm not sure it's relevant. Please give me your opinion, I'll change it if necessary.
- Added media queries with breakpoints at 320px, 568px and 768px. 
I think that's it.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './globalScss/index.scss';
-import App from './pages/App/App';
+// import App from './pages/App/App';
+import UnsupportedFileFormat from './components/UnsupportedFileFormat/UnsupportedFileFormat.component';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<UnsupportedFileFormat />, document.getElementById('root'));
```
Comment (ArnasDickus) [inline src/index.js:7]: I don't think you should add an Unsupported file format in index.js. It should be put as a component in-app.js

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, so I was expecting to have an unsupported file format. Popup appear on the page. Right now it looks more as a page.

So the problem is that this project doesn't have a SPA routing [react routing](https://trello.com/c/iMs6MLFn/102-add-react-routing). So player bar and visualizer in app.js is not the correct place. However, until when let's leave it as it is.

Here's my suggestions:
1) Add unsupported file format component in app.js
2) Remove the background since this popup is going to appear in the center while the user is trying to import the song.
3) Connect it with player bar add music button. If the format is not supported this popup should appear.

**Comment by lucierabahi** [comment]
Yes, it should be only the /pop-upbox component replacing the "upload song" box, on the same page.
Index.js was for testing my component while developing, forgot to move it to app.js before making the PR, sorry.
Ok for background, ok for player bar. 
Thank you for the review.

**Comment by ArnasDickus** [comment]
Testing travis.

**Pull request closed without merging**